plugins/plugin-sass/plugin.js

```diff
@@ -71,9 +71,9 @@ module.exports = function sassPlugin(_, {native, compilerOptions = {}} = {}) {
       // check no ext: "_index" (/a/b/c/foo/_index)
       this._markImportersAsChanged(filePathNoExt);
       // check no underscore: "index.scss" (/a/b/c/foo/index.scss)
-      this._markImportersAsChanged(filePath.replace(/\/_(.+)(?!\/)/, '/$1'));
+      this._markImportersAsChanged(filePath.replace(/\/_(.+)(?!\/\\)/, '/$1'));
       // check no ext, no underscore: "index" (/a/b/c/foo/index)
-      this._markImportersAsChanged(filePathNoExt.replace(/\/_(.+)(?!\/)/, '/$1'));
+      this._markImportersAsChanged(filePathNoExt.replace(/\/_(.+)(?!\/\\)/, '/$1'));
```

Comment:
This doesn’t seem to be working for me? I think the negative lookahead is looking for a `/` followed by a `\`, not OR?

Reply:
ah you're right

Reply:
I know it’s lamer than this RegEx but I find it easier-to-follow to`.split(path.sep)`, only remove the `_` prefix on the last segment. Then you can `.join(path.sep)`

Reply:
Shoot just saw the update. I think you also need to change `replace(/\/_` to `replace(/[\/\\]_` because it’s not paying attention to backslashes

Reply:
grrrrrrrrrr

Reply:
this is a pretty hot code path, so I think the perf gain is worth it here. But usually yea, I'd agree

Reply:
I don’t know much about it but I don’t think `split()` / `join()` isn’t significantly slower? If we’re concerned about perf here

https://www.measurethat.net/Benchmarks/Show/1557/0/regex-vs-splitjoin

But to be clear I’m more concerned about code maintenance than perf.